lib/schema.js

```diff
@@ -269,21 +285,43 @@ class Metaschema {
   buildCategory(category, ...args) {
     return this.categories.get(category).factory(...args);
   }
+
+  [initActions]() {
+    for (const [categoryName, { definition: category }] of this.categories) {
+      this.actions.set(
+        categoryName,
+        iter(Object.keys(category))
+          .filter(key => extractDecorator(category[key]) === 'Action')
+          .map(key => {
+            const action = {
+              categoryName,
+              name: key,
+              definition: category[key],
+              form: this.forms.get(`${categoryName}.${key}`),
+            };
+
+            return [key, action];
+          }).collectTo(Map)
+      );
+    }
+  }
 }
 
 // Creates Metaschema instance
 //   schemas - <Iterable> schemas in form [name, schema, source]
 //             (the 'source' is optional)
-// Returns: <Metaschema>
+// Returns: [<MetaschemaError>, <Metaschema>]
```

Comment:
I don't think we support such notation though =).

Reply:
Well, we probably should.

Reply:
IIUC we will as soon as https://github.com/metarhia/metaschema/pull/107 lands.

Reply:
@SemenchenkoVitaliy can you confirm that such notation will be supported?
Also, what about:
```js
// Returns: <Array>
//   <MetaschemaError>
//   <Metaschema>
```
Will it be possible to write it that way?

Reply:
### Creates Metaschema instance
`create(schemas)`
  - `schemas` - `<Iterable>` schemas in form [name, schema, source]
        (the 'source' is optional)

Returns: `[ <MetaschemaError>, <Metaschema> ]`

This is the part of generated md file with corrected comments.

> Also, what about:
> 
> ```js
> // Returns: <Array>
> //   <MetaschemaError>
> //   <Metaschema>
> ```
> 
> Will it be possible to write it that way?

Yes, this is totally possible and correct, though we should choose one so we will have same comments style across all files. @tshemsedinov @belochub @nechaido @lundibundi WDYT?